scripts/kata-install/osc-kata-install.sh

```diff
@@ -143,11 +196,14 @@ install() {
 	# Clean up temp dir
 	rm -rf /host/tmp/extensions/
 
+	# Copy configs
+	copy_kata_remote_config_files
```

Comment:
If the goal is to simplify the workflow, the same overall simplification could be achieved by simply invoking the config script, no?